SearchTests/TestHelpers.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  TestHelpers.swift
+//  Routing
```

Comment:
Header comment is wrong. We should probably put this stuff somewhere common so we could include it as a git submodule or something.